Doc/library/sys.monitoring.rst

```diff
@@ -292,9 +292,10 @@ in Python (see :ref:`c-api-monitoring`).
    Activates all the local events for *code* which are set in *event_set*.
    Raises a :exc:`ValueError` if *tool_id* is not in use.
 
-Local events add to global events, but do not mask them.
-In other words, all global events will trigger for a code object,
-regardless of the local events.
+Local events add to global events. In other words, all global events
```

Comment:
Why not leave in the 'but do not mask them' wording?

Reply:
Because I thought I was communicating the same thing in my rewrite, but apparently not 😅.

I personally found the "mask" part confusing as that suggests to me there's two firings as local events don't mask global events since they "add" together. But that's not what happens, it's really a toggle for the event that can be flipped on by either locally or globally and it's not an XOR or something.